stock_inventory_verification_request/models/stock_inventory.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class StockInventory(models.Model):
+    _inherit = 'stock.inventory'
+
+    requested_verification = fields.Boolean(
+        string='Requested Verification?', default=False, copy=False)
```

Comment:
`default=False` is [explicitely ignored on Boolean fields](https://github.com/odoo/odoo/blob/10.0/odoo/models.py#L2289-L2290).